include/cppkafka/utils/roundrobin_poll_adapter.h

```diff
@@ -173,25 +174,39 @@ class RoundRobinPollAdapter
      */
     MessageList poll_batch(size_t max_batch_size, std::chrono::milliseconds timeout);
     
+    /**
+     * \brief Gets the number of assigned partitions that can be polled across all topics
+     *
+     * \return The number of partitions
+     */
+    size_t get_num_partitions();
```

Comment:
Wouldn't it be better to allow getting the current assignment? That is, the list of all topic+partitions.

Reply:
My initial intention is to eventually allow polling for all (default) or certain topics only by adding a setter function. In the case of certain topics, then this function will not be identical to get current assignment.

Reply:
I mean, you always have a list of queues you're pulling from. Whatever your list of queues is should be accessible from the outside, regardless of whether they came through a partition assignment or it was the user who set them.